tests/test_check_new_assets.py

```diff
@@ -11,21 +11,31 @@
 
 @pytest.mark.parametrize("chain", get_substrate_chains(), ids=task_ids)
 class TestAssets:
-    # workaround to differences between asset names at runtime and in our configuration
-    asset_mapping = {
-        'AUSD': 'KUSD',
-        'XX': 'xx'
-    }
-    
+    # assets that have no working cases or having differences between
+    # asset names at runtime and in our configuration
+    exclusions = {
+        'Karura': {'AUSD': 'KUSD'},
+        'Acala': {'AUSD': 'KUSD'},
+        'Shiden': {'AUSD': 'KUSD'},
+        'Bifrost Kusama': {'AUSD': 'KUSD', 'DOT': ''},
+        'Kintsugi': {'AUSD': 'KUSD', 'INTR': '', 'iBTC': 'IBTC', 'DOT': ''},
+        'Basilisk': {'AUSD': 'KUSD'},
+        'Turing': {'AUSD': 'KUSD'},
+        'XX network': {'XX': 'xx'}}
+
     def test_has_new_assets(self, chain: Chain):
-        
+
         chain_assets = {asset['symbol'].upper(): '' for asset in chain.assets}
-        chain_assets.update(self.asset_mapping)
+        if chain.name in self.exclusions.keys():
+            for key, value in self.exclusions.items():
+                if key == chain.name:
+                    chain_assets.update(value)
```

Comment:
Dict comprehension here may decrease number of lines and improve readability:
```suggestion
        chain_assets.update(self.asset_mapping)
        
        if chain.name in self.exclusions:
            chain_assets.update({ex_asset: '' for ex_asset in self.exclusions[chain.name]})
```